azurelinuxagent/common/protocol/extensions_goal_state_from_vm_settings.py

```diff
@@ -202,6 +202,7 @@ def _parse_agent_manifests(self, vm_settings):
         #     "gaFamilies": [
         #         {
         #             "name": "Prod",
+        #             "version": "9.9.9.9",
```

Comment:
Since you are doing it prod, it's good to add this for test if present.

Reply:
I specifically didn't add it in `Test` to keep an example of the case where this property might not be present in all GAFamilies :)